src/Particles/babylon.particleSystem.ts

```diff
@@ -59,7 +59,16 @@
 
         public customShader: any = null;
         public preventAutoStart: boolean = false;
-        
+
+        public cellWidth: number;
+        public cellHeight: number;
+        public cellIndex: number;

```

Comment:
This should be a property of the particle only no?

Reply:
this property is used to set the cellIndex in the system it self ex 
var particleSystem = new BABYLON.ParticleSystem("particles", 2000, scene, null, 64);
particleSystem.particleTexture = new BABYLON.Texture("textures/player.png", scene, true,false, BABYLON.Texture.TRILINEAR_SAMPLINGMODE);;
particleSystem.cellIndex = 44;

Reply:
ok so it could be called particleSystem.startCellIndex
Alongside particleSystem.endCellIndex